dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/notification/UserNotificationServiceBean.java

```diff
@@ -76,23 +82,52 @@ public void delete(UserNotification userNotification) {
         em.remove(em.merge(userNotification));
     }
 
-    public void sendNotification(AuthenticatedUser dataverseUser, Timestamp sendDate, Type type, Long objectId) {
-        sendNotification(dataverseUser, sendDate, type, objectId, "");
+    public void sendNotificationWithoutEmail(AuthenticatedUser dataverseUser, Timestamp sendDate, NotificationType type) {
+        UserNotification userNotification = new UserNotification();
+        userNotification.setUser(dataverseUser);
+        userNotification.setSendDate(sendDate);
+        userNotification.setType(type);
+
+        save(userNotification);
     }
 
-    public void sendNotification(AuthenticatedUser dataverseUser, Timestamp sendDate, Type type, Long objectId, String comment) {
-        sendNotification(dataverseUser, sendDate, type, objectId, comment, null);
+    public void sendNotification(AuthenticatedUser dataverseUser,
```

Comment:
Wouldn't it be a little clearer if we have methods `sendNotification`, `sendEmail` (private?) and `sendNotifiacationWith/AndEmail`

Reply:
Since I already began refactoring this in next Async mails task, I would leave this section for another code review.